src/index.tsx

```diff
@@ -3,6 +3,8 @@ import { createRoot } from 'react-dom/client';
 import 'wicg-inert';
 import { registerSW } from 'virtual:pwa-register';
 
+import '#src/modules/register';
```

Comment:
I would rather export a default method and actually call it, instead of relying on side effects of an import.